### DISCUSSION TITLE
common peaks for multiome merge/integration of different samples?

### DISCUSSION THREAD

**Discussion by kathirij** [stuart-lab/signac#1143] [Q&A] [answered]
Hi,
   I have three multiome samples (10x platform) that are all somewhat related but with different experimental perturbations leading to high degree of diversity in cell populations. My general question is how to merge and integrate the datasets? I will have a total of ~25K cells once all samples are merged. I have gone through several discussions on this topic here... All have been super useful and fruitful. 

However, specifically, I want to clarify two points and one additional request:
1. I want to know what peaks are unique to each of the samples... Will using disjoin be better than reduce? Will that allow me to find overlapping vs unique peaks downstream or am I just interpreting the need for finding unified peaks incorrectly?
2. I also want to know what is the consensus (or closest to it anyway) on merging/integrating multimodal datasets that are not biological replicates and therefore should have a high amount of heterogeneity? Are the normalization step described in issue #827, the overall approach for using harmony to avoid overcorrecting rna data in #438 still the best/latest practices?
3. Lastly, if anyone who has used the solution provided in #1012 can provide their code (with whatever details you are comfortable with), it will be very much appreciated...

Thank you for this wonderful resource!

**Comment by timoast** [comment, marked as answer]
> I want to know what peaks are unique to each of the samples... Will using disjoin be better than reduce? Will that allow me to find overlapping vs unique peaks downstream or am I just interpreting the need for finding unified peaks incorrectly?

I would still use reduce, when running disjoin you can end up with a lot of very small ranges due to the way disjoin works. If you prefer to use disjoin, you probably want to filter out ranges below a certain width before running downstream steps. In comparing across conditions, you first want to define one peak set that you're going to quantify across all the samples (example shown in the [merge vignette](https://satijalab.org/signac/articles/merging.html)), and then perform differential accessibility testing between groups of cells. I wouldn't rely on presence/absence of a peak call in different samples as a method of assessing differential accessibility.

> I also want to know what is the consensus (or closest to it anyway) on merging/integrating multimodal datasets that are not biological replicates and therefore should have a high amount of heterogeneity? Are the normalization step described in issue https://github.com/timoast/signac/discussions/827, the overall approach for using harmony to avoid overcorrecting rna data in https://github.com/timoast/signac/discussions/438 still the best/latest practices?

The goal of doing integration is to identify the equivalent states across datasets in an unsupervised analysis. Once you've done that, you always want to go back to the original (uncorrected) data for comparing expression levels or accessibility levels across cells. The problem of integration "erasing" biological heterogeneity only really occurs if we incorrectly align cells that originate from distinct biological states. You should do some checks after integration (regardless of the method used) to convince yourself that the results make sense (eg, aligned cells from different batches still express the same canonical cell type markers). In this sense, whether you want to use Harmony or the Seurat integration method or other methods does not matter.
